### PR TITLE
[#709] column-break mixin

### PR DESCRIPTION
**Pull request by nickcooley**
Introducing column-break... Comments added to columns.scss.  Tests included.

### CODE REVIEW

**Comment by chriseppstein** [comment]
The compass convention is to have a 1-1 mapping between compass css3 mixins and css3 properties. I'm not opposed to _also_ having a short cut mixin that makes common use cases more convenient.

File: frameworks/compass/stylesheets/compass/css3/_columns.scss [hunk 1]
```diff
@@ -58,3 +58,36 @@
     -moz, -webkit, -o, not -ms, not -khtml, official
   );
 }
+
+ 
+// Mixin for setting column breaks.
+//
+// * legal values for '$type' : 'before', 'after', 'inside'
```
Comment (chriseppstein) [inline frameworks/compass/stylesheets/compass/css3/_columns.scss]: doesn't seem like these would need to be quoted.

**Comment by nickcooley** [comment]
no prob.  Will update.

**Comment by scottdavis** [comment]
this will need to be rebased to master it will not cleanly merge in either stable or master

**Comment by nickcooley** [comment]
So what do I need to do?

**Comment by scottdavis** [comment]
rebase can be a pain in the butt but your commits should be ok
heres the best i can find on short notice

http://gitready.com/intermediate/2009/01/31/intro-to-rebase.html
http://mettadore.com/analysis/a-simple-git-rebase-workflow-explained/

**Comment by nickcooley** [comment]
Thanks Scott --

I'll see what I can do.  I did a revert and didn't realize it would wreak this kind of havoc on the commit.

**Comment by nickcooley** [comment]
currently rebasing -- please pay no attention to the additional commits as they will be removed.  Sorry for the sloppiness...

**Comment by scottdavis** [comment]
np as long as it gets in sync with master

**Comment by nickcooley** [comment]
Hey, @scottdavis -- I rebased and got my repo up to speed with master (sorry for the confusion there, was working off an old version that hadn't been updated/my bad).   What I noticed, though, was that when I pulled latest from repo, I also got [ef267dd] included as part of the commit (It's Gemfile.lock)  Let me know if you want me to back that out (and how), or if you'll just cherry-pick that.  Thanks for your help and suggestions today!

**Comment by nickcooley** [comment]
Hey @scottdavis -- Is there anything else you need from me on this?

**Comment by nickcooley** [comment]
Hey @scottdavis @ericam -- bump on this.   Any issues here?

**Comment by chriseppstein** [comment]
Sorry, I've been super busy but hoping to plow thru some PRs soon. Unless @scottdavis beats me to it.

**Comment by nickcooley** [comment]
Thanks @chriseppstein  -- Heard about the great talk you gave in Florida last week.  I was speaking about Sass/Compass in London at the same time.  

I've been a little wary of the rebase that I did considering all the additional commits (the ones at the top) that came along with it (just fetched latest from base,)  so please let me know if you need me to do anything here.  I'm a little new to github, so if there's anything you need me to change about the PR -- I'm glad to.

**Comment by chriseppstein** [comment]
I pulled the columns and hyphenation commits into stable for the 0.12.2 release  as commits 0cdff5ae9a54dea94aa9c781020b67723cfe0cf0 and 8db0b40a65002aa1ddca3adb0f24302093ca5571 respectively.

**Comment by nickcooley** [comment]
Thanks Chris!  Will look at other issues to resolve!

**Pull request closed without merging**